src/_pytest/_code/code.py

```diff
@@ -1044,6 +1042,8 @@ def toterminal(self, tw) -> None:
             for line in self.lines:
                 red = line.startswith("E   ")
                 tw.line(line, bold=True, red=red)
+            if self.reprlocals:
+                self.reprlocals.toterminal(tw, indent=" " * 8)
```

Comment:
I think this should have consistent indentation as with the case below, i.e. none currently.
(but when adding indentation it might be better to have 4 here (and below then probably also) maybe)

Reply:
I believe 8 is better here, as the source code is intended to 4, which makes it harder to visually scan it.